src/utils/helpers/list/buildQueryVariables.ts

```diff
@@ -30,10 +30,13 @@ const mapSortFields: any = {
 // ----
 
 interface PaginationValues {
-  numberToFetch: number
-  paginationOffset: number
+  numberToFetch?: number
+  paginationOffset?: number
 }
 
-export function getPaginationVariables(page: number, perPage = 20): PaginationValues {
-  return { numberToFetch: perPage, paginationOffset: (page - 1) * perPage }
+export function getPaginationVariables(
+  page: number,
+  perPage: number | undefined
+): PaginationValues {
+  return { numberToFetch: perPage, paginationOffset: perPage ? (page - 1) * perPage : undefined }
```

Comment:
This will cause a conflict with the fix I've made in my filtering PRs to fix the pagination bug you found the other week. 

I guess it's probably easiest to just leave it like this, and I'll have to make a slight modification to mine before merging it.

Reply:
Yep. I think it's probably nice to have the "Get ALL" solution as well